.github/workflows/terraform-security.yaml

```
@@ -0,0 +1,33 @@
+name: Security (Terraform)
```

```suggestion
name: Trivy
```

Can we also rename the fileame to trivy.yaml?

Again: Trivy does much more than terraform.  It checks helm charts / kubernetes yaml, as well as dependencies for a ton of different ecosystems.  If we want to to scope this to terraform, we should disable all the non-terraform trivy check.  I'd recommend we implement it in an ecosystem agnostic way though -- I'm happy to get reports about whatever trivy wants to check.

---

Sure -- this also seems like reasonable naming. I tried to set `scan-type: config` to keep Trivy's focus away from dependencies and whatnot, but it could still alert on non-TF stuff.